scripts/graph.sh

```diff
@@ -26,8 +26,8 @@ dot -Tpng <packages-graph.dot >"$DIR"/../packages-graph.png
 dot -Tsvg <packages-graph.dot >"$DIR"/../packages-graph.svg
 
 if acyclic packages-graph.dot | dot -Tcanon >packages-graph-sans-cycles.dot; then
-    echo "No cycles in 'dependencies' of packages."
+    echo 1>&2 "No cycles in 'dependencies' of packages."
 else
-    echo "Cycles detected. These lines appear only in the original graph and not the acyclic variant:"
+    echo 1>&2 "Cycles detected. These lines appear only in the original graph and not the acyclic variant:"
```

Comment:
kinda weird to have stderr say "these:" referring to stdout because a user could end up seeing one stream and not the other. Consider reverting.

Reply:
I see the commit message rationale now, that stdout should have parseable output. That's persuasive to me.. If this script was going to live longer I'd say to update the message to not assume both streams will be seen at once, but I don't really think it's worth bothering

Reply:
It's a cheap fix, and I have to make some other little doc changes, so I'll update the message.